contributors/help-wanted.md

```diff
@@ -0,0 +1,138 @@
+
+All `good first issue` items need to follow the guidelines for `help wanted`
+items in addition to meeting the following criteria:
+
+- **No Barrier to Entry**
+  The task is something that a new contributor can tackle without advanced
+  setup or domain knowledge.
+
+- **Solution Explained**
+  The recommended solution is clearly described in the issue.
+
+- **Provides Context**
+  If background knowledge is required, this should be explicitly mentioned and a
+  list of suggested readings included.
+
+- **Gives Examples**
+  Link to examples of similar implementations so new contributors have a
+  reference guide for their changes.
+
+- **Identifies Relevant Code**
+  The relevant code and tests to be changed should be linked in the issue.
+
+- **Ready to Test**
+  There should be existing tests that can be modified, or existing test cases
+  fit to be copied. If the area of code doesn't have tests, before labeling the
+  issue, add a test fixture. This prep often makes a great `help wanted` task!
+
+<!-- TODO find a good one
+A good example of a `good first issue` description can be found here: [kubernetes/kubernetes#68231](https://github.com/kubernetes/kubernetes/issues/68231).
+-->
+
+These commands can be used in the GitHub issue comments to control the `good first issue` label:
+
+- `/good-first-issue` : Adds the `good first issue` label to an issue. Also adds
+  the `help wanted` label, if not already present.
+- `/remove-good-first-issue` : Removes the `good first issue` label from an issue.
+
+## Suggestions for Experienced Community Members
+
+We encourage our more experienced members to help new contributors, so that the
+KubeVirt community can continue to grow and maintain the kind, inclusive
+community that we all enjoy today.
+
+The following suggestions go a long way toward preventing "drive-by" PRs, and
+ensure that our investment in new contributors is rewarded by returning contributors.
+
+- Provide extra assistance during reviews on `good first issue` pull requests.
+- Answer questions and identify useful docs.
+- Offer advice such as "One way to reproduce this in a cluster is to do X and
+  then you can use kubectl to poke around," or "Did you know that you can
+  use fake clients to setup and test this easier?"
+- Help new contributors learn enough about the project, setting up their
+  environment, running tests, and navigating this area of the code so that they
+  can tackle a related `help wanted` issue next time.
+
+If you make someone feel like a part of our community, they will know that it is safe to ask
+questions, that people will let them know the rules, and that their
+contributions are helpful and appreciated. They will stick around! 🌈
+
+- Encourage new contributors to seek help on the appropriate slack channels,
+  introduce them, and include them in your conversations.
+- Invite them to the SIG meetings.
+- Give credit to new contributors so that others get to know them: "Hey, would
```

Comment:
Is that applicable to KubeVirt? We only require a single LGTM?

Reply:
We do require a single LGTM, but since the OWNERS changes we now might required multiple approvals.

Reply:
understood now, adjusted.